pom.xml

```diff
@@ -55,6 +55,12 @@
       <version>${akka.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
pls remember about `gradle.build`.

Reply:
What do you mean by `gradle.build` as I don't see a gradle build system in sumobot?

Reply:
Right, sorry, confused it with bender.